libfwupdplugin/fu-context.h

```diff
@@ -54,6 +54,15 @@ typedef void (*FuContextLookupIter)(FuContext *self,
  **/
 #define FU_CONTEXT_FLAG_SAVE_EVENTS (1u << 0)
 
+/**
+ * FU_CONTEXT_FLAG_SAVE_REMOVED_DEVICES:
+ *
+ * Save removed devices so that they can be replayed to emulate devices.
+ *
+ * Since: 1.8.10
```

Comment:
can you make this 1.8.11 please, i'll tag a release today of 1.8.11 and we can merge after that is tagged.

Reply:
Yes, done